src/components/chips/js/chipsDirective.js

```diff
@@ -121,7 +123,7 @@
       <md-chips-wrap\
           ng-if="!$mdChipsCtrl.readonly || $mdChipsCtrl.items.length > 0"\
           ng-keydown="$mdChipsCtrl.chipKeydown($event)"\
-          ng-class="{ \'md-focused\': $mdChipsCtrl.hasFocus(), \'md-readonly\': !$mdChipsCtrl.ngModelCtrl }"\
+          ng-class="{ \'md-focused\': $mdChipsCtrl.hasFocus(), \'md-readonly\': !$mdChipsCtrl.ngModelCtrl, \'md-not-removeable\': !$mdChipsCtrl.removeable}"\
```

Comment:
Why not just `md-removable` here instead of `md-not-removable`? In the CSS, you should be able to use `:not(.md-removable)` which I think makes a bit more sense.